content/faq.md

```diff
@@ -0,0 +1,72 @@
+---
+title: FAQ
+menu:
+  nav:
+    name: help
+    url: "/faq/"
+    weight: 100
+---
+
+## Purpose
+
+### What is this sandbox?
+
+We wanted to make it easier for developers to get started with playing around and using our version of Cloud Foundry. Since most developers don't want to install an entire Kubernetes cluster just to try out a platform, it made more sense to bring it to you! 
+
+Whether you are an experienced developer or brand new to the idea, we hope this sandbox can be place to try random ideas and see if they work. Share with us what you build or ask for help on our Discord!
+
+### What can I build in this sandbox?
+
+Anything! This getting started guide has some very simple examples, but you can host very complex applications in Cloud Foundry.
+
+If you want some suggestions on what to build, join us on our discord to see what others are building!
+
+### What languages are supported?
+
+We support the build packs for: 
+ - Java
+ - Ruby
+ - Node.js
+ - Golang
+ - Python
+ - Php
+ - .Net Core
+ - Static Files
+ - and binary applications
+
+ If you are interested in helping expand this list, let us know!
+
+## Rules and Quotas
+
+### How many Applications can I run?
+10
+
+With a total of 2GB of requested ram.
+
+### How many services can I use from the marketplace?
+
+4, This should give you enough to try each of the services.
+
+### Can I do everything that is available in Cloud Foundry?
+
+No, this is specifically a *Developer* Sandbox. We are not allowing access to a large chunk of functionality such as:
+
+ - Auto-Scaling
+ - Custom Buildpacks
+ - User and Team Management
+ - Blue/Green Deployments
+
+If there are features that you 
+
+### Can I install my own Open Service Broker? 
+
+Talk to us, if it's something that we think would be interesting to the whole community, maybe.
+
+### Can I route non-HTTP routes. 
+
+Yes, this is a feature of Cloud Foundry, but we have it turned off by default.
```

Comment:
Add: Let us know if this limiting you, we're happy to look into ways to support your use case.